packages/cli/src/CommandsLogic.ts

```diff
@@ -321,6 +333,70 @@ export class CommandsLogic {
     }
   }
 
+  async withdrawToOwner (options: WithdrawOptions): Promise<WithdrawalResult> {
+    const transactions: string[] = []
+    try {
+      console.log('Withdrawing from GSN relayer to owner')
+      const relayManager = options.keyManager.getAddress(0)
+      console.log('relayManager is', relayManager)
+      const relayHub = await this.contractInteractor._createRelayHub(options.config.relayHubAddress)
+      const stakeManagerAddress = await relayHub.stakeManager()
+      const stakeManager = await this.contractInteractor._createStakeManager(stakeManagerAddress)
+      const { owner } = await stakeManager.getStakeInfo(relayManager)
+      if (owner.toLowerCase() !== options.config.ownerAddress.toLowerCase()) {
+        throw new Error(`Owner in relayHub ${owner} is different than in server config ${options.config.ownerAddress}`)
+      }
+
+      const balance = await relayHub.balanceOf(relayManager)
+      console.log('manager hub balance is', balance.toString())
+      if (balance.lt(options.withdrawAmount)) {
+        throw new Error(`Relay manager hub balance ${balance.toString()} lower than withdrawalAmount`)
+      }
+      const method = relayHub.contract.methods.withdraw(options.withdrawAmount, owner)
+      const encodedCall = method.encodeABI()
+      console.log('encodedCall', encodedCall)
+      const nonce = await this.contractInteractor.getTransactionCount(relayManager)
+      const gasPrice = parseInt(await this.contractInteractor.getGasPrice())
+      const gasLimit = 1e5
+      console.log('gasPrice, nonce', gasPrice, nonce)
+      const txToSign = new Transaction({
+        to: options.config.relayHubAddress,
+        value: 0,
+        gasLimit,
+        gasPrice,
+        data: Buffer.from(encodedCall.slice(2), 'hex'),
+        nonce
+      }, this.contractInteractor.getRawTxOptions())
+      console.log('Calling in view mode')
+      await method.call({
+        from: relayManager,
+        to: options.config.relayHubAddress,
+        value: 0,
+        gas: gasLimit,
+        gasPrice
+      })
+      console.log('Signing tx', txToSign.toJSON())
+      const signedTx = options.keyManager.signTransaction(relayManager, txToSign)
+      console.log(`signed withdrawal hex tx: ${signedTx.rawTx}`)
+      if (options.broadcast) {
```

Comment:
what is the purpose of NOT sending the signed TX? I think its very confusing. might be needed for testing (that is default to broadcast,)

Reply:
To test the tx in view mode only, and get the tx details in case you want to broadcast it yourself. I think that it's useful to run it only in view mode, to see the tx details, since using it on mainnet can have consequences...